src/posts/experimental-static-build.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 'Scaling Astro Sites to Thousands of Pages'
+description: 'A new experimental flag in Astro build enables building sites with tens of thousands of pages.'
+publishDate: 'January 25, 2022'
+authors:
+  - matthew
+lang: 'en'
+---
+
+Astro's about to get a lot faster with a newly built build system intended to scale to tens, or even hundreds, of thousands of pages. If you hang out in our [Discord](https://astro.build/chat) or pay attention to recently releases you might have seen a lot of discussion about a "static build". This is our new implementation of `astro build` that does 2 things:
+
+- Improves build times by up to 75%.
```

Comment:
What's the real world impact? I'd love something like "... up to 75%. A build that previously took 3 minutes should now take about 45 seconds." Stats based on a 75% reduction, but you might have real numbers.

Reply:
It depends on usage. If you are doing something like reading from an API (Pokemon demo) to generate pages you will get 75%. If you have lots of markdown files in your project you might get a smaller improvement. And if you have a really small project (like only a few pages) it might even be slower (2.5 seconds instead of 2 seconds, for example). So we can't commit to always having a certain build improvement, it depends on the project.